File: python/questions/moveZeroes.py
from typing import List


class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        zp = 0
        for i in range(len(nums)):
            if nums[i] != 0:
                nums[zp], nums[i] = nums[i], nums[zp]
                zp += 1

File: typescript/questions/moveZeroes.ts
function moveZeroes(nums: number[]): void {
  let nl = nums.length;
  let zp = 0;
  for (let i = 0; i < nl; ++i) {
    if (nums[i] != 0) {
      [nums[zp], nums[i]] = [nums[i], nums[zp]];
      ++zp;
    }
  }
}
